### DISCUSSION TITLE
Arito - Practice Math Skills

### DISCUSSION THREAD

**Discussion by 3xp11i** [prakhartiwari0/Arito#59] [General]
Adding [giscus](https://giscus.app/) commenting system on the homepage of Arito!
Now everybody could share their review for the App right on the homepage!😇

**Comment by 3xp11i** [comment]
I hope you enjoy this app, do share your review after using it.

**Comment by lvnilesh** [comment]
Hi Prakhar.

**Reply by 3xp11i** [reply]
Hey Nilesh! Great to see you here. Do share your review after using the app :)

**Comment by venkataraviraja5** [comment]
Hello world

**Comment by Sonikm** [comment]
Amazing

**Comment by Nayan808** [comment]
good one to go

**Comment by sayande717** [comment]
> Very hard **addition** & **substraction**: 👶
> Very hard **multiplication** & **division**: 💀

But jokes apart, it's amazing! The fact that you also derived your own formulae just for it makes it extra cool 😊

**Reply by 3xp11i** [reply]
Haha, nice one!
Thanks for your feedback :smile: 
Glad you enjoyed it...

**Comment by siddhant-khisty** [comment]
Great UI. The math stuff goes over my head tho. Need a calculator for it

**Reply by 3xp11i** [reply]
Thanks! wish you the best for maths practice :)

**Comment by chthonn** [comment]
I enjoyed this web app a lot.

**Reply by 3xp11i** [reply]
Glad to know that you enjoyed it!
